src/http/error_handler.ts

```diff
@@ -0,0 +1,21 @@
+import { Errors, IErrorHandler, Response } from "../../mod.ts";
+
+export class ErrorHandler implements IErrorHandler {
+  /**
+   * Catch and handle the given error.
+   *
+   * @param error - The Error object that is thrown during runtime.
+   * @param request - The original request that threw the error. Note this is
+   * not that `Drash.Request` object. Reason being, the `Drash.Request` object
+   * can also throw an error and would not be available as a parameter.
+   * @param response - The `Drash.Response` object.
+   */
+  public catch(
+    error: Errors.HttpError,
+    _request: Request,
+    response: Response,
+  ): void {
+    response.status = error.code ?? 500;
```

Comment:
```typescript
if (error instanceof Errors.HttpError) {
  response.status = error.code;
} else {
  response.status = 500;
}
```

would be better or not ?
what happens if developer thrown a simple Error and not an HttpError ?

Reply:
I think yes, to work correctly with simple Error